src/Controller/Admin/AdminUnregistrationController.php

```diff
@@ -43,12 +43,9 @@ public function exportUnregistrationsAction(): Response
      * @Method("GET")
      * @Security("has_role('ROLE_ADMIN_UNREGISTRATIONS')")
      */
-    public function exportUnregistrationsPartialAction(Request $request): Response
+    public function exportUnregistrationsPartialAction(Request $request, UnregistrationRepository $repository): Response
     {
-        $page = $request->query->get('page', 1);
-
-        $manager = $this->getDoctrine()->getManager();
-        $unregistrations = $manager->getRepository(Unregistration::class)->findPaginatedForExport($page, self::PER_PAGE);
+        $unregistrations = $repository->findPaginatedForExport($request->query->get('page', 1), self::PER_PAGE);
```

Comment:
it should be `$request->query->getInt()` here.
if I request this path with `?page=dimitri`, i would get a 500 because method `findPaginatedForExport` is typehinter with `integer`

Reply:
agree, fixed 👍